test/functional/api/buffer_spec.lua

```diff
@@ -1,5 +1,5 @@
 -- Sanity checks for buffer_* API calls via msgpack-rpc
-local helpers = require('test.functional.helpers')
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
Maybe you already checked this, but are we sure we can't just set this globally in busted, instead of having to add this to every file ?